<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="ln_ans">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>640</y>
      <width>961</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>650</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Answer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>700</y>
      <width>961</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>710</y>
      <width>79</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>79</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>String 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>79</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>String 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>100</y>
      <width>79</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_str1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>341</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_str2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spb_N">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>128</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>1071</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_memcpy">
       <property name="text">
        <string>memcpy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_memmove">
       <property name="text">
        <string>memmove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_strcpy">
       <property name="text">
        <string>strcpy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_strncpy">
       <property name="text">
        <string>strncpy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_strcat">
       <property name="text">
        <string>strcat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>1071</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_memset">
       <property name="text">
        <string>memset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_strtok">
       <property name="text">
        <string>strtok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_strcoll">
       <property name="text">
        <string>strcoll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_strxfrm">
       <property name="text">
        <string>strxfrm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_strlen">
       <property name="text">
        <string>strlen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>1071</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_strcmp">
       <property name="text">
        <string>strcmp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_strncmp">
       <property name="text">
        <string>strncmp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_memcmp">
       <property name="text">
        <string>memcmp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_strncat">
       <property name="text">
        <string>strncat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
